### PR TITLE
Add systemd service file

### PR DESCRIPTION
**Pull request by Rjasuja**
Adding systemd service file to support systemd initialization manager.

Issue: #36
Test: Verified service on Ubuntu

Signed-off-by: Ritul Jasuja <ritul.jasuja@intel.com>

### CODE REVIEW

**Comment by otaviobp** [comment]
the patch looks good to me, but I would like to test it in Aero before merging it.

**Comment by lucasdemarchi** [comment]
@Rjasuja See changes I mentioned in mavlink-router and apply here

**Comment by Rjasuja** [comment]
@lucasdemarchi i applied the changes mentioned for mavlink-router. This PR is with those changes

**Comment by lucasdemarchi** [comment]
When I applied on mavlink-router I did more changes. Check my comment there. It needs to pass CI btw

File: configure.ac [hunk 1]
```diff
@@ -84,6 +101,9 @@ AC_CONFIG_FILES([
 ])
 
 #####################################################################
+AC_SUBST(SYSTEMD_SYSTEMUNITDIR, [${path_systemunitdir}])
+
+AC_SUBST(SYSTEMD_USERUNITDIR, [${path_userunitdir}])
```
Comment (lucasdemarchi) [inline configure.ac]: This line should go away

**Comment by Rjasuja** [comment]
Canceling this because of new PR

**Pull request closed without merging**